resources/install.rb

```diff
@@ -0,0 +1,71 @@
+# resources/install.rb
+
+# Resource for InfluxDB install
+
+property :arch_type, kind_of: String, default: 'amd64'
+property :include_repository, kind_of: [TrueClass, FalseClass], default: true
+property :influxdb_key, kind_of: String, default: 'https://repos.influxdata.com/influxdb.key'
+property :install_type, kind_of: String, default: 'package'
+default_action :install
+
+action :install do
+  case install_type
+  when 'package'
+    if node.platform_family? 'rhel'
+      yum_repository 'influxdb' do
+        description 'InfluxDB Repository - RHEL \$releasever'
+        baseurl node['influxdb']['upstream_repository']
+        gpgkey influxdb_key
+        only_if { include_repository }
+      end
+    else
+      package 'apt-transport-https' do
+        action :install
```

Comment:
Is this still necessary?  Shouldn't the influxdb binary already be self-contained?